lesson17/script.js

```diff
@@ -1,47 +1,31 @@
 const slider = document.getElementById("js-slider");
 
-const createElement = ({tagName,id,className,text}) => {
-    const element = document.createElement(tagName);
-    if(id){
-        element.id = id;
+const createElement = (obj) => {
+    const element = document.createElement(obj.tagName);
+    if(obj.id){
+        element.id = obj.id;
     }
-    element.className = className;
-    element.textContent = text;
+    element.className = obj.className;
+    element.textContent = obj.text;
+    element.src = obj.src;
+    element.alt = obj.alt;
     return element;
 }
```

Comment:
I deleted createImage and createButton,
Changed the createElement function generically.